client/src/components/Reviews.jsx

```diff
@@ -50,10 +58,14 @@ class Reviews extends React.Component {
   }
 
   render() {
+
     return (
       <div className="Reviews">
-        <div> Hello, This is Reviews section!</div>
-        <ReviewsCustomerImages />
+        {/* <ErrorBoundary> */}
+        <ReviewsCustomerImages reviews={this.state.reviews} />
+        {/* </ErrorBoundary> */}
+        {/* <ReviewsPopup /> */}
```

Comment:
Consider removing commented out code for production.

Reply:
Agree, will remove comments for next PR. Wanted to implement Error boundaries but its in progress.